scale/docs/rest/v6/job_type.rst

```diff
@@ -410,6 +411,8 @@ Response: 200 OK
 | max_scheduled            | Integer           | Maximum  number of jobs of this type that may be scheduled to run at the |
 |                          |                   | same time. May be null.                                                  |
 +--------------------------+-------------------+--------------------------------------------------------------------------+
+| max_tries                | Integer           | Number of times this a job of this type will be automatically retried.   |
```

Comment:
minor typo. I think you meant "Number of times a job of this type will be automatically retried."

Reply:
Good eye. Fixed with better wording.